docs/TESTS.md

```diff
@@ -1,112 +1,65 @@
-## Xcode Tutorial
+## Testing Tutorial
 
-This guide explains how to create and configure a Swift-based Xcode project for Exercism.
+This guide explains how to run unit tests for Exercism Swift exercises, and optionally how to use Xcode for development.
 
-Apple does not support user-defined project templates in Xcode, and the default templates are more elaborate than those needed for Exercism. This guide bridges that gap.
+Exercism uses [Swift Package Manager](https://github.com/apple/swift-package-manager/tree/master/Documentation) to package files and tests for Swift.
```

Comment:
I agree with you Bob. The issue here is that while xcode is an excellent IDE, it is only avalible on the mac. In order to be more inclusive we should strive give more options. When Swift was mac only it made sense to focus on xcode since this was the only way to work with swift. Learning xcode was part of learning Swift. This is still the case for the Objective-C track.  Do you have any specific changes you want to suggest? I know that you spent some time editing the current version. An alternative would be to pull the xcode detail instructions into a blog post that we could then link in here.